en/news/_posts/2024-07-16-dos-rexml-cve-2024-39908.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: news_post
+title: "CVE-2024-39908 : DoS in REXML"
+author: "watson1978"
+translator:
+date: 2024-07-16 03:00:00 +0000
+tags: security
+lang: en
+---
+
+There is a DoS vulnerability in REXML gem. This vulnerability has been assigned the CVE identifier [CVE-2024-39908](https://www.cve.org/CVERecord?id=CVE-2024-399086). We strongly recommend upgrading the REXML gem.
```

Comment:
Link points to `CVE-2024-399086`. This should be `CVE-2024-39908`.

Reply:
Thanks. I sent a PR to fix typo (https://github.com/ruby/www.ruby-lang.org/pull/3318)